attributes/default.rb

```diff
@@ -47,3 +47,6 @@
     omnibus['build_user_password'] = '$1$QTCj0tQy$C60hWNmo8wZo.ctvDSy9p/'
   end
 end
+
+# Chef-installed bento boxes default 7-zip here. Make sure it gets in the path.
+default['7-zip']['home']    = "#{ENV['SYSTEMDRIVE']}\\Program Files\\7-zip"
```

Comment:
@jkeiser Perhaps the proper fix is to update the default value in the 7-zip cookbook?
https://github.com/sneal/7-zip/blob/master/attributes/default.rb#L31

I also see this open PR: https://github.com/sneal/7-zip/pull/2